library/Icingadb/Widget/Detail/ObjectDetail.php

```diff
@@ -70,7 +71,9 @@ protected function createActions()
 
         foreach ($this->compatObject->getActionUrls() as $i => $url) {
             $navigation->addItem(
-                sprintf(t('Action %d'), $i + 1),
+                (string) new Icon('forward', [
+                    'title' => t('Link opens in new window')
+                ]) . (string) new Text(" " . $url),
```

Comment:
Maybe we should only show the url without query here. Urls can get very long and the path itself is usually the most interesting part anyway. So please only show the url path in the label.